cfme/tests/automate/test_domain.py

```diff
@@ -182,8 +182,10 @@ def test_domain_name_wrong(appliance):
         initialEstimate: 1/60h
         tags: automate
     """
-    with pytest.raises(Exception, match='Name may contain only'):
-        appliance.collections.domains.create(name='with space')
+    view = navigate_to(appliance.collections.domains, 'Add')
+    view.name.fill('Dummy Domain')
+    view.add_button.click()
+    view.flash.assert_message('Name may contain only alphanumeric and _ . - $ characters')
```

Comment:
**Required** I think it would be good to extend this even more and verify that the `Dummy Domain` was not added even though a flash error appeared by checking the all view.

Reply:
@jawatts 
According to me, 
There are two ways to check if ```dummy domain``` is exists or not.
1. By instantiating domain object with name: ```dummy domain```
```
wrong_domain = domain.instantiate(name=wrong_domain)
assert not wrong_domain.exists
```
2. Checking in list of domains on all page
```
view = navigate_to(domain, 'All')
all_domains = view.domains.read()
for i in range(len(all_domains)):
     assert wrong_domain != all_domains[i]['Name']
```
 I have updated PR with first way. 
Please let me know, if it needs more changes.

Reply:
Option 1 here is good by me, thanks for adding that in